pkg/controller/clusterconnection/clusterconnection_controller.go

```diff
@@ -99,6 +99,7 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 	for _, secretName := range []string{
 		render.PacketCaptureServerCert,
 		monitor.PrometheusServerTLSSecretName,
+		"guardian" + certificatemanagement.TrustedCertConfigMapSuffix,
```

Comment:
It's worth having a helper for this so we can ensure the same value is always used.

```
func CertBundleName("guardian") string {
}
```

would be used here as well as in the certificatemanagement code when generating the bundle.

Reply:
done.